### ISSUE TITLE
Encourage others to use pelita and add them to a list of users

### ISSUE THREAD

**Issue by Debilski** [ASPP/pelita#381]
Should probably add something to the README that they tell us and we can warn them if we break master.

**Comment by otizonaizit** [comment]
>Should probably add something to the README that they tell us and we 
>can warn them if we break master.

Eh? We do *not* break master, ever, right? Not anymore, at least ;-)

As soon as @DoriekeG is ready with the new a_star/bfs implementation, we 
will release 1.0 and follow strict backward-compatibility rules, 
including the tournament script :-)

**Comment by Debilski** [comment]
And then we’ll have several branches for 1.0/2.0? I remember we had that once and eventually even I myself saw that it was a bad idea. ;)

**Comment by otizonaizit** [comment]
>And then we’ll have several branches for 1.0/2.0? I remember we had 
>that once and eventually even I myself saw that it was a bad idea. ;)

No, we will have master and will merge to master whatever is mature 
enough to be merged. When there is enough stuff, or stuff is critical, 
we'll make a release. No maintenance branches: two external users are 
not enough to justify it, especially since we have one single maintainer ;-)

**Comment by Debilski** [comment]
Every day is a new day and every commit is a new release. :)

**Comment by otizonaizit** [comment]
No, that is exactly the point. If we make explicit numbered release, this means that we can tell to people that they install release vX.X and be done with it. This releases wil have WHAT_IS_NEW and CHANGELOG, will be available on PyPI for installation with pip, etc. 

Master is then used to preview the new release, but shouldn't be used in production, because we may still decide to change stuff before releasing, new APIs may be removed if they appear to be flawed, etc.

Really, it is time to take the burden of some responsibility towards (potential) users... pelita is not a ruby gem...

**Comment by Debilski** [comment]
Ah, that‘s what you mean. I tend to forget about PyPI for us…

And I’m unsure how it should work. Where should the pelitagame and tk_viewer.py and module_player.py files end up? Should they all land in /usr/bin per default with their stupid names? (And there’s tournament.py and maybe even others.)

**Comment by Debilski** [comment]
#383

**Comment by otizonaizit** [comment]
>And I’m unsure how it should work. Where should the pelitagame and 
>tk_viewer.py and module_player.py files end up? Should they all land in 
>/usr/bin per default with their stupid names? (And there’s 
>tournament.py and maybe even others.)

pelitagame should go to /usr/bin (but then it could be called pelita). 
The tournament would then be /usr/bin/pelita_tournament or something. 
for all other scripts that are just of internal usage, like 
tk_viewer.py, they will go somewhere where standard Python libraries put 
them, most probably /usr/lib/pelita/scripts/tk_viewer.py . But this will 
be decided by setup.py if we set it up correctly.
